lib/topological_inventory/persister/ar_helper.rb

```diff
@@ -2,20 +2,38 @@ module TopologicalInventory
   module Persister
     module ArHelper
       def self.load_environment!
+        ENV["RAILS_ENV"] ||= "development"
+
+        database_yaml_path = root.join("config", "database.yml")
+        database_yaml = YAML.load_file(database_yaml_path) if database_yaml_path.exist?
```

Comment:
Need to require yaml here and we can probably remove this now https://github.com/ManageIQ/topological_inventory-core/blob/5dc481e54a98ebdf0a8109237efb91b700bec88d/Rakefile#L23